libs/ngx-localstorage/src/lib/utils.ts

```diff
@@ -32,10 +32,11 @@ export const setProperty = (path: string[] | string, value: any, object: any, fa
 /**
  * Constructs the storage key based on a prefix - if given - and the key itself
  */
-export const constructKey = (key: string, prefix?: string, configuredPrefix?: string): string => {
+export const constructKey = (key: string, prefix?: string, configuredPrefix?: string, delimiter?: string): string => {
   const prefixToUse = prefix || configuredPrefix;
+  const delimietrToUse = delimiter || '_';
```

Comment:
When using a default value defined in the `defaultConfig` you don't have to handle it here and just pass it through

Reply:
Fixed